linux/tools/mock_feedback_server.py

```
@@ -0,0 +1,63 @@
+            }
+            with lock:
+                with requests_file.open("a", encoding="utf-8") as handle:
+                    handle.write(json.dumps(record, ensure_ascii=False, sort_keys=True) + "\n")
+            response = json.dumps({"ok": True, "received": len(body)}, separators=(",", ":")).encode("utf-8")
+            self.send_response(200)
+            self.send_header("Content-Type", "application/json")
+            self.send_header("Content-Length", str(len(response)))
+            self.end_headers()
+            self.wfile.write(response)
+
+        def log_message(self, format: str, *args: object) -> None:
```

Ruff (0.15.12)

[error] Function argument `format` is shadowing a Python builtin

(A002)